CMakeLists.txt

```diff
@@ -50,6 +50,13 @@ pkg_check_modules(LIBXC REQUIRED IMPORTED_TARGET libxc)
 if(NOT TARGET PkgConfig::LIBINT)
   pkg_check_modules(LIBINT REQUIRED IMPORTED_TARGET libint2>=2.6)
 endif()
+find_package(ecpint REQUIRED)
```

Comment:
Not right now, but later (next week?), we should required version 1.5 here.

Reply:
1.0.5? Actually we should require it now, because all the tests segfault otherwise.

Reply:
Well, fedora and opensuse didn't receive the 1.0.5 package yet. https://bodhi.fedoraproject.org/updates/FEDORA-2021-5464f657f7 will need 7 more days.

Reply:
But Fedora has 1.0.4 with your patch.

Reply:
Ok, there is another bug in libecpint, that prevents from exporting the right version in the cmake target, so there is nothing we can right now.

Reply:
We can required 1.0.6 once https://github.com/robashaw/libecpint/pull/32 is through. ;-)

Reply:
okay